<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaPrincipal</class>
 <widget class="QMainWindow" name="TelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instale Já!</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/LogoInstaleJá.png</normaloff>:/images/LogoInstaleJá.png</iconset>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Buscar Cliente por Nome: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineInBuscaNome"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeClientes">
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>50</number>
      </attribute>
      <column>
       <property name="text">
        <string>Id Cliente/Servico</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome Cliente/Tipo Serviço</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Telefone Cliente/Data Serviço</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Endereço Cliente/Preço Serviço</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>OBS</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnTotal">
        <property name="text">
         <string>Calcular Total</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTotalMes">
        <property name="text">
         <string>Calcular Total no Mês</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOp_es">
    <property name="title">
     <string>Opções</string>
    </property>
    <addaction name="actionCriarOS"/>
   </widget>
   <addaction name="menuOp_es"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCriarOS">
   <property name="text">
    <string>Criar OS</string>
   </property>
   <property name="statusTip">
    <string>Criar uma Ordem de Serviço</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
